### ISSUE TITLE
Is my training&testing process correct?

### ISSUE THREAD

**Issue by JifanYin** [VDIGPKU/QGCN#2]
@ljwdust Thank you for your impressive work, and I trained my model with following script:
`python src/main.py --dataloader jpeg_qm --size 64 --batchSize 16 --quality [1,60] --colorMode RGB --net qmar --n_resblock 64 --nEpochs 600 --loss L1 --cuda --threads 4 --gpus [0] --savedir DIV2K_1_60`

While testing, I used the model saved with minimum average loss and ran:
`python src/eval_qm.py --model ../checkpoint/DIV2K_1_60/Epoch_83_loss_0.05982.pth --GT --gtdata BSDS500 --quality 10 --save BSDS500_10 --cuda`

The results turned to be:

> 【Average of all images:】
> PSNR: LQ: 25.6168, HQ: 25.0371, Increment: -0.5797
> SSIM: LQ: 0.7497, HQ: 0.7048, Increment: -0.0449
> PSNR-B: LQ: 20.6690, HQ: 23.4099, Increment: 2.7410
> All time: 228.7779s, primary time: 10.6682s

I found that in your paper(Table II), the results on testset BSDS500 with Quality=10 is better than I tested above. It may be something wrong happen?

Environment:
pytorch 0.4.1
cuda 10.2
Tesla P100
